Client/Frontend/Browser/NoImageModeHelper.swift

```diff
@@ -34,18 +34,22 @@ class NoImageModeHelper: TabHelper {
     }
 
     static func isNoImageModeAvailable() -> Bool {
-        return AppConstants.MOZ_NO_IMAGE_MODE
+        if #available(iOS 11, *) {
+            return AppConstants.MOZ_NO_IMAGE_MODE
+        }
+        return false
     }
 
     static func isActivated(_ prefs: Prefs) -> Bool {
         return prefs.boolForKey(NoImageModePrefsKey.NoImageModeStatus) ?? false
     }
 
     static func toggle(profile: Profile, tabManager: TabManager) {
-        let enabled = isActivated(profile.prefs)
-
-        profile.prefs.setBool(!enabled, forKey: PrefsKeys.KeyNoImageModeStatus)
-        tabManager.tabs.forEach { $0.setNoImageMode(!enabled, force: true) }
-        tabManager.selectedTab?.reload()
+        if #available(iOS 11, *) {
+            let enabled = isActivated(profile.prefs)
+            profile.prefs.setBool(!enabled, forKey: PrefsKeys.KeyNoImageModeStatus)
+            tabManager.tabs.forEach { $0.noImageMode = $0.noImageMode }
```

Comment:
Wouldn't this be a no-op? I would think it should be setting `noImageMode` to the value of `enabled`?

Reply:
oops, good catch!